Echo.Core/src/Evaluation/Operations/HilbertCurvePattern.cs

```diff
@@ -0,0 +1,101 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using CodeHelpers.Packed;
+
+namespace Echo.Core.Evaluation.Operations;
+
+public class HilbertCurvePattern : ITilePattern
+{
+	public Int2[] CreateSequence(Int2 size)
+	{
+		var result = size.X > size.Y ?
+			Hilbert2D(Int2.Zero, new Int2(size.X, 0), new Int2(0, size.Y)).ToArray() :
+			Hilbert2D(Int2.Zero, new Int2(0, size.Y), new Int2(size.X, 0)).ToArray();
+
+		// flip first half of the array to make it start from the middle
```

Comment:
This flip post process works well; I really like the checkerboard pattern (it allows for seeing the results much faster). However, can we try to make the pattern expand out in all four directions rather than only two? Currently, this pattern first moves right then wraps back left. It would be much better if it goes from the center towards all four corners.